include/mcu.hpp

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file mcu.hpp
+ *
+ * @brief MCU related
+ */
+
+#ifndef MCU_HPP
+#define MCU_HPP
+
+#include <cstdint>
+
+/*****************************************
+ * Public Function Prototypes
+ *****************************************/
+
+extern "C" {
+/**
+ * @brief Initializes System Clock.
+ * @note  Defined by cube.
+ */
+void SystemClock_Config(void);
+}
+
+namespace hal {
+class mcu {
+public:
+    /**
+     * @brief Initializes MCU and some peripherals.
+     */
+    static void init();
+
+    /**
+     * @brief Put the MCU to sleep.
+     *
+     * @param ms  Sleep time in milliseconds
+     */
+    static void sleep(uint32_t ms);
+};
+};  // namespace hal
```

Comment:
```suggestion
/**
 * @brief Initializes System Clock
 * @note  Defined by cube
 */
void SystemClock_Config(void);
}

namespace hal {
/**
 * @brief Microcontroller unit class
 */
class mcu {
public:
    /**
     * @brief Initializes MCU and some peripherals
     */
    static void init();

    /**
     * @brief Put the MCU to sleep
     *
     * @param ms  Sleep time in milliseconds
     */
    static void sleep(uint32_t ms);
};
};  // namespace hal
```

só pra dedixar no padrao kk

Reply:
(tirei os pontos finais dos comentarios kkkkk)

Reply:
e botei um comentario na classe

Reply:
Sou fã de ponto final em tds comentários.

Reply:
impossivel KKKKKKK

Reply:
> Sou fã de ponto final em tds comentários.

https://github.com/Team-Micras/MicrasFirmware/commit/b8c93fe4c5c6850fe46feb5b1a042cba827ca939

Reply:
a hipocrisia

Reply:
impossivel kkkkkkkkkk

Reply:
> Sou fã de ponto final em tds comentários.

https://github.com/Team-Micras/MicrasFirmware/commit/b8c93fe4c5c6850fe46feb5b1a042cba827ca939

Reply:
a hipocrisia

Reply:
https://github.com/Team-Micras/MicrasFirmware/pull/28

Sou um homem mudado.

Reply:
kkkkkkkkkkk